.git-commit-template

```diff
@@ -0,0 +1,18 @@
+# Summarize the change
+# |<----   Preferably using up to 50 chars   --->|<------------------->|
+
+# (Optional) Explain why this change is being made
+# |<----   Try to limit each line to a maximum of 72 characters   ---->|
+
+# To include this change in changelog, uncomment and fill in the following:
+# = changelog =
+# msg:
+# type:
+# resolves:
+# related:
+
+# msg        = message to be included in changelog
+# type       = one of: bugfix/enhancement/security
+# resolves   = URLs to bugs or issues resolved by this commit
+# related    = any related bugs or issues
```

Comment:
I wouldn't, because then you can't simply uncomment the line and add the text, but you also have to delete the text that is already there. I tried to make it a comment in the following way, but it only works when the `#` is at the beginning of the line.
```
# = changelog =
# msg:         # message to be included in changelog
# type:        # one of bugfix/enhancement/security
# resolves:    # URLs to bugs or issues resolved by this commit
# related:     # any related bugs or issues
```

Reply:
Okay. In vim it's basically the same amount of key strokes either way, but I guess not everyone is enjoying the blissful ease of editing text files that vim offers, for whatever reason :sunglasses: We can keep it the way it is...

Reply:
Yeah, and now that [nano is going to be default in Fedora](https://fedoraproject.org/wiki/Changes/UseNanoByDefault)... ;)

Reply:
Hehe.. I always get annoyed when I have to use nano, as I don't know how to use it and it seems to be so cumbersome :grinning: